tools/WebJobs.Script.Performance/WebJobs.Script.Performance.App/PerformanceManager.cs

```diff
@@ -40,48 +39,29 @@ public PerformanceManager()
             _client = new ComputeManagementClient(credentials);
             _client.SubscriptionId = Settings.SiteSubscriptionId;
 
-            _tests = new List<TestDefinition>()
-            {
-                new TestDefinition()
-                {
-                    FileName = "win-csharp-ping.jmx",
-                    Desciption = "C# Ping",
-                    Runtime = "dotnet"
-                },
-                new TestDefinition()
-                {
-                    FileName = "win-js-ping.jmx",
-                    Desciption = "JS Ping",
-                    Runtime = "node"
-                },
-                new TestDefinition()
-                {
-                    FileName = "win-java-ping.jmx",
-                    Desciption = "Java Ping",
-                    Runtime = "java"
-                }
-            };
+            _tests = new List<TestDefinition>();
+            Settings.Tests.Bind(_tests);
         }
 
         public async Task ExecuteAsync(string testId)
         {
             // We assume first word in testId is platform
-            var test = _tests.FirstOrDefault(x => x.FileName.ToLower() == testId.ToLower());
+            var test = _tests.FirstOrDefault(x => x.Jmx.ToLower() == testId.ToLower());
```

Comment:
Can we just change this to perform a string.Equals with a case insensitive comparison?